### PR TITLE
add new calendar: China

### PR DESCRIPTION
**Pull request by iamsk**
refs #

For information, read and make sure you're okay with the [CONTRIBUTING document](https://github.com/novafloss/workalendar/blob/master/CONTRIBUTING.rst#adding-new-calendars).

- [x] Tests with a significant number of years to be tested for your calendar.
- [x] Docstrings for the Calendar class and specific methods.
- [x] Calendar country / label added to the README.rst file,
- [x] Changelog amended with a mention like: "Added ``<country>`` by ``@pseudo`` (#)"

### CODE REVIEW

**Comment by iamsk** [comment]
@brunobord is this ok for merging?

File: .travis.yml [hunk 1]
```diff
@@ -13,9 +13,6 @@ matrix:
     - python: 2.7
       env: TOXENV=py27
 
-    - python: 3.4
```
Comment (brunobord) [inline .travis.yml]: I dislike the fact that this PR introduces the dropping of Python 3.4. I see in other jobs that there's an issue with `pandas` ; so I'll try to find a way to solve this incompatibility more gracefully, and see if this sunsetting of Python 3.4 is necessary.
I'll keep you posted.

File: Changelog.md [hunk 2]
```diff
@@ -4,6 +4,12 @@
 
 Nothing here yet.
 
+## v3.2.2 (2018-12-25)
```
Comment (brunobord) [inline Changelog.md]: Please do not put new version numbers and/or release dates here.
You should rather add the "new calendars" section and your changelog item to the "master" list.
When it'll be ready, I'll make the proper release, etc, don't worry.

**Comment by brunobord** [comment]
All right, so master is now Python 3.4 - ready, so you can rebase your branch from master (or you may checkout the upstream repository, and cherry-pick your commits and make another pull-request if you prefer).
You would pay attention not to remove the py34 job environment **and** edit the changelog file in accordance to my latest remarks.

Thank you for your nice contribution, we're almost there! :o)

File: .travis.yml (hunk 1 above)
Comment (iamsk) [reply .travis.yml]: @brunobord good job!

File: Changelog.md (hunk 2 above)
Comment (iamsk) [reply Changelog.md]: all right

**Comment by brunobord** [comment]
Looking good, now. No offense, but I'm going to retrieve your branch locally, squash commits in order to have a cleaner history. Don't worry, your work on this calendar is still ACE and I can't thank you enough for this HUGE bump in the global population that will be able to use workalendar!

**Pull request closed without merging**